seqr/views/utils/test_utils.py

```diff
@@ -224,7 +224,7 @@ def _check_login(self, url, permission_level, request_data=None):
         'genotypes': {
             'NA19675': {
                 'sampleId': 'NA19675',
-                'ab': 0.7021276595744681,
+                'ab': 0.702127659574,
```

Comment:
The reason for removing 4 decimals is Python 2 has lower accuracy and the expected data has fewer decimals than this data. Python 3 supports more decimals so that the expected result becomes different from that of Python 2. To make the expected data the same for both Python version, reduce some digits here.